presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -14,55 +14,35 @@
 
 package com.facebook.presto.spi.statistics;
 
-import java.util.HashMap;
-import java.util.Map;
-
 import static com.facebook.presto.spi.statistics.Estimate.unknownValue;
-import static java.util.Collections.unmodifiableMap;
 import static java.util.Objects.requireNonNull;
 
 public final class ColumnStatistics
 {
-    private final Map<String, Estimate> statistics;
-    private static final String DATA_SIZE_STATISTIC_KEY = "data_size";
-    private static final String NULLS_COUNT_STATISTIC_KEY = "nulls_count";
-    private static final String DISTINCT_VALUES_STATITIC_KEY = "distinct_values_count";
+    private final Estimate dataSize;
+    private final Estimate nullsCount;
+    private final Estimate distinctValuesCount;
 
     private ColumnStatistics(Estimate dataSize, Estimate nullsCount, Estimate distinctValuesCount)
     {
-        requireNonNull(dataSize, "dataSize can not be null");
-        requireNonNull(nullsCount, "nullsCount can not be null");
-        requireNonNull(distinctValuesCount, "distinctValuesCount can not be null");
-        statistics = createStatisticsMap(dataSize, nullsCount, distinctValuesCount);
-    }
-
-    private static Map<String, Estimate> createStatisticsMap(Estimate dataSize, Estimate nullsCount, Estimate distinctValuesCount)
-    {
-        Map<String, Estimate> statistics = new HashMap<>();
-        statistics.put(DATA_SIZE_STATISTIC_KEY, dataSize);
-        statistics.put(NULLS_COUNT_STATISTIC_KEY, nullsCount);
-        statistics.put(DISTINCT_VALUES_STATITIC_KEY, distinctValuesCount);
-        return unmodifiableMap(statistics);
+        this.dataSize = requireNonNull(dataSize, "dataSize can not be null");
+        this.nullsCount = requireNonNull(nullsCount, "nullsCount can not be null");
+        this.distinctValuesCount = requireNonNull(distinctValuesCount, "distinctValuesCount can not be null");
     }
 
     public Estimate getDataSize()
     {
-        return statistics.get(DATA_SIZE_STATISTIC_KEY);
+        return dataSize;
     }
 
     public Estimate getNullsCount()
     {
-        return statistics.get(NULLS_COUNT_STATISTIC_KEY);
+        return nullsCount;
     }
 
     public Estimate getDistinctValuesCount()
     {
-        return statistics.get(DISTINCT_VALUES_STATITIC_KEY);
-    }
-
-    public Map<String, Estimate> getStatistics()
```

Comment:
It breaks backward-compatibility. Maybe we should leave for a while as deprecated?